examples/playground/json-examples/tagmap.json

```diff
@@ -12,32 +12,23 @@
     {
       "@@type": "MapView",
       "controller": true,
-      "mapStyle": "https://rivulet-zhang.github.io/dataRepo/mapbox/style/map-style-dark-v9-no-labels.json"
+      "mapStyle": "https://basemaps.cartocdn.com/gl/dark-matter-nolabels-gl-style/style.json"
     }
   ],
   "layers": [
-    {
-      "@@type": "TagmapLayer",
```

Comment:
This has not worked for some time now. `TagmapLayer` is not defined in the JSON configuration. We could certainly import the class but honestly I think it will be very confusing given that `TagmapLayer` is not documented anywhere.